pandas/core/generic.py

```diff
@@ -8900,14 +8900,24 @@ def _where(
         inplace=False,
         axis=None,
         level=None,
-        errors="raise",
+        errors=lib.no_default,
     ):
         """
         Equivalent to public method `where`, except that `other` is not
         applied as a function even if callable. Used in __setitem__.
         """
         inplace = validate_bool_kwarg(inplace, "inplace")
 
+        if errors is not lib.no_default:
```

Comment:
any reason not to use the `deprecate_kwarg` decorator?

Reply:
i think with the decorator the warning message would be about _where instead of where/mask

Reply:
I assume that would need to use the decorator on all 4 of the public methods instead of just once here?

Reply:
either way works, yah